air/src/preparation_step/preparation.rs

```diff
@@ -134,14 +139,16 @@ fn make_exec_ctx(
     signature_store: SignatureStore,
     run_parameters: &RunParameters,
 ) -> PreparationResult<ExecutionCtx<'static>> {
-    let call_results = measure!(
+    let call_results_js = measure!(
         CallResultsRepr
             .deserialize(call_results)
             .map_err(|e| PreparationError::call_results_de_failed(call_results.clone(), e))?,
         tracing::Level::INFO,
         "CallResultsRepr.deserialize",
     );
 
+    let call_results = unjs_call_results(call_results_js).map_err(PreparationError::InvalidCallResultId)?;
```

Comment:
I think that one day we can abstract from `u32` in call results, we can also use Strings ot Vec<u8> for them if it's more convenient b/c for the host side it's just a tag which internal structure is known only to AquaVM.

Reply:
Well, we may query call ID by its string representation, I thought about it.

Reply:
Switched to strings for call IDs.